piker/pp.py

```diff
@@ -285,15 +285,14 @@ def minimize_clears(
         all transactions before the last net zero size to avoid
         unecessary history irrelevant to the current pp state.
 
-
         '''
-        size: float = 0
+        size: float = self.size
```

Comment:
This and the subtract change below were the main issue with getting the `pps.toml` clears tables per position correct..